### PR TITLE
Various function implementations from LFS

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

**Comment by no92** [comment]
The ABI checker complaining is intended and should not result in an ABI break.

File: options/linux/include/sys/swap.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#ifndef _SYS_SWAP_H
+#define _SYS_SWAP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define	SWAP_FLAG_PREFER		0x8000
```
Comment (Geertiebear) [inline options/linux/include/sys/swap.h:8]: What happened to the formatting here?

File: options/glibc/include/sys/user.h [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #ifndef _SYS_USER_H
 #define _SYS_USER_H
 
+#include <mlibc/arch-defs.h>
```
Comment (avdgrinten) [inline options/glibc/include/sys/user.h:4]: `mlibc/` headers must not be included from public headers. Use `bits/` instead.

File: options/linux/include/sys/swap.h [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#define SWAP_FLAG_DISCARD 0x10000
+
+int swapon(const char *, int);
+int swapoff(const char *);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif // _SYS_SWAP_H
```
Comment (Dennisbonke) [inline options/linux/include/sys/swap.h:20]: We should use C style comments here.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
LGTM, one comment.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**